packages/validators/rollup.config.mjs

```diff
@@ -0,0 +1,28 @@
+import { nodeResolve } from '@rollup/plugin-node-resolve';
+import typescript from '@rollup/plugin-typescript';
+import dts from 'rollup-plugin-dts';
+export default [
+  {
+    input: 'src/index.ts',
+    output: [
+      {
+        dir: 'dist',
+        format: 'esm',
+        sourcemap: true,
+      },
+    ],
+    plugins: [nodeResolve(), typescript({ tsconfig: './tsconfig.json' })],
```

Comment:
Ely added this new dev-specific config a few days ago that appears to be missing here.
```suggestion
    plugins: [
      nodeResolve(),
      typescript({ tsconfig: './tsconfig.json', noEmitOnError: process.env.DEV ? false : true }),
    ],
```